localization/english/religion/wc_core_tenets_l_english.yml

```
@@ -121,4 +121,6 @@
  tenet_bloody_feast_desc:0 "The most valuable sacrifice we can offer is the sacrifice of sentient life. Blood goes to the divine while the flesh can be consumed to absorb the wisdom and strength of the deceased."
 
  special_doctrine_legions_puppet_name:0 "$game_concept_legions_puppet$"
- special_doctrine_legions_puppet_desc:0 "Our master [obey_legion|E] for now and thus we follow, but nothing can withstand death."
\ No newline at end of file
+ special_doctrine_legions_puppet_desc:0 "Our master [obey_legion|E] for now and thus we follow, but nothing can withstand death."
+ 
+ doctrine_parameter_hostility_override_tenet_dharmic_pacifism:0 "Considers and is considered by [faiths|E] with Chi Pacifism to be [GetHostilityLevelName( FAITH_DOCTRINE_PARAMETER.GetIntValue )|V]"
```

To have same localization pattern as in gnostic(vanilla)/eastern(mod) hostility overrides
```suggestion
 doctrine_parameter_hostility_override_tenet_dharmic_pacifism:0 "Considers other Chi Pacifistic [faiths|E] to be [GetHostilityLevelName( FAITH_DOCTRINE_PARAMETER.GetIntValue )|V]"
```